JASP-Engine/JASP/po/R-nl.po

```diff
@@ -4472,7 +4472,7 @@ msgid "Scatter plots"
 msgstr "Spreidingsdiagrammen"
 
 msgid "Correlation plot"
-msgstr "Correlatie Grafiek"
+msgstr "Correlatiegrafiek"
```

Comment:
`Correlatiediagram` om consistent te zijn met de andere aanpassingen

Reply:
Klopt idd. Is aangepast.